.github/workflows/rust.yml

```
@@ -80,7 +81,32 @@ jobs:
         uses: actions-rs/cargo@v1
         with:
           command: test
-          args: --all --no-fail-fast
+          args: --doc
```

I think this should probably revert to the previous, right?
```suggestion
          args: --all --no-fail-fast
```